src/e2e/java/teammates/e2e/pageobjects/StudentProfilePage.java

```diff
@@ -159,12 +159,21 @@ public void showPictureEditor() {
         waitForUploadEditModalVisible();
     }
 
-    public void verifyPhotoSize(String height, String width) {
-        assertEquals(height, browser.driver.findElement(By.className("profile-pic")).getCssValue("height"));
-        assertEquals(width, browser.driver.findElement(By.className("profile-pic")).getCssValue("width"));
+    public void closePictureEditor() {
         click(uploadEditModal.findElement(By.className("close")));
     }
 
+    public void verifyPhotoSize(int height, int width) {
+        String browserHeight = browser.driver.findElement(By.className("profile-pic")).getCssValue("height");
+        String browserWidth = browser.driver.findElement(By.className("profile-pic")).getCssValue("width");
+        //Remove px from Dimension Strings to turn it into a number
```

Comment:
This is not necessary; the code reads exactly what this comment says (i.e. comment doesn't value-add). That's the advantage of using good abstractions.